lib/coophub/backends/backends.ex

```diff
@@ -158,6 +151,9 @@ defmodule Coophub.Backends do
 
       {:error, %HTTPoison.Error{reason: reason}} ->
         {:error, reason}
+
+      _ ->
+        {:error, nil}
```

Comment:
Error swallowing here? Should return `{:error, reason}` to be logged from its caller.

Reply:
good catch! that was added for test purposes in the development phase!